core/server/worker/src/main/java/alluxio/worker/block/DefaultBlockWorker.java

```diff
@@ -296,7 +296,8 @@ public void createBlockRemote(long sessionId, long blockId, String tierAlias, lo
   @Override
   public BlockReader readUfsBlock(long sessionId, long blockId, long offset)
       throws BlockDoesNotExistException, IOException {
-    return mUnderFileSystemBlockStore.createBlockReader(sessionId, blockId, offset);
+    return mBlockStore.createBlockReader(sessionId, blockId, offset);
+    //return mUnderFileSystemBlockStore.createBlockReader(sessionId, blockId, offset);
```

Comment:
If not needed, we can remove it instead of comment out.

Reply:
The comment is deleted.